ign_rviz_plugins/src/rviz/plugins/CovarianceVisual.cpp

```diff
@@ -0,0 +1,519 @@
+// Copyright (c) 2022 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ignition/rviz/plugins/CovarianceVisual.hpp"
+
+// diagonalization utilities
+namespace
+{
+double deg2rad(double degrees)
+{
+  return degrees * 3.14159265358979 / 180.0;
+}
```

Comment:
remove this and use `IGN_RTOD` and `IGN_DTOR` from `ign-math`. `Angle` class defines them